bugwarrior/command.py

```diff
@@ -15,6 +16,7 @@
 )
 
 import logging
+logging.basicConfig()
```

Comment:
Oh I see. It's run by `bugwarrior.config.validate_config` which isn't called until the end of `bugwarrior.config.load_config` -- *after* the exception you're catching. Rather than set up logging twice I think we need to move the logging setup in `validate_config` to earlier to the entry point.